src/cli/project/export/conda_explicit_spec.rs

```diff
@@ -0,0 +1,232 @@
+use std::collections::HashSet;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use clap::Parser;
+
+use crate::cli::cli_config::PrefixUpdateConfig;
+use crate::cli::LockFileUsageArgs;
+use crate::lock_file::UpdateLockFileOptions;
+use crate::Project;
+use rattler_conda_types::{ExplicitEnvironmentEntry, ExplicitEnvironmentSpec, Platform};
+use rattler_lock::{CondaPackage, Environment, Package};
+
+#[derive(Debug, Parser)]
+#[clap(arg_required_else_help = false)]
+pub struct Args {
+    /// Output directory for rendered explicit environment spec files
+    pub output_dir: PathBuf,
+
+    /// Environment to render. Can be repeated for multiple envs. Defaults to all environments
+    #[arg(short, long)]
+    pub environment: Option<Vec<String>>,
+
+    /// The platform to render. Can be repeated for multiple platforms.
+    /// Defaults to all platforms available for selected environments.
+    #[arg(short, long)]
+    pub platform: Option<Vec<Platform>>,
+
+    /// PyPI dependencies are not supported in the conda explicit spec file.
+    /// This flag allows creating the spec file even if PyPI dependencies are present.
+    #[arg(long, default_value = "false")]
+    pub ignore_pypi_errors: bool,
```

Comment:
I think I commented about this earlier but I can t find my previous comment. I feel this flag is somewhat ambiguous. Its not ignoring errors, its not throwing an error if there are any pypi dependencies. I feel the name is a bit misleading. 

Also given that the command is already explicitly requesting an explicit conda environment spec I feel we can simply ignore pypi dependencies without erroring. Perhaps at most a warning?

Do you have an opinion @ruben-arts ?

Reply:
The way I have it written (which is based on `pixi-pack`), if there are pypi dependencies in the lockfile for that platform/env, it called `miette::bail!`. I think this makes sense since we want to make it very clear to the user that the conda env produced by export is going to be incomplete vs what was in pixi if there are pypi dependencies, and they knowingly have to ignore that error. I'm happy to change it though if you think it's ambiguous.

Reply:
I have no strong feelings about this. Ill let @ruben-arts decide but lets merge on Monday if he doesn't reply. I think this has been sitting still for a bit too long!